views/item.ejs

```diff
@@ -1,135 +1,129 @@
-        Object.values = function values(O) {
-        	return reduce(keys(O), (v, k) => concat(v, typeof k === 'string' && isEnumerable(O, k) ? [O[k]] : []), [])
+        // polyfill
+        if (!Object.values) {
+          const reduce = Function.bind.call(Function.call, Array.prototype.reduce)
+          const isEnumerable = Function.bind.call(Function.call, Object.prototype.propertyIsEnumerable)
+          const concat = Function.bind.call(Function.call, Array.prototype.concat)
+          const keys = Reflect.ownKeys
+          Object.values = function values(O) {
+            return reduce(keys(O), (v, k) => concat(v, typeof k === 'string' && isEnumerable(O, k) ? [O[k]] : []), [])
+          }
         }
-      }
       </script>
-      <button
-        class='btn btn-primary'
-        id='saveButton'
-        onclick='handleEditClick()'
-        type='button'
-      >
+      <button class='btn btn-primary' id='saveButton' onclick='handleEditClick()' type='button'>
         Save
       </button>
 
       <script>
-      function handleDeleteClick (event) {
-        event.preventDefault()
-        fetch(document.location.pathname, {
-          method: 'delete'
-        }).then((response) => {
-          if (response.ok) {
-            window.location = `/tables/<%= TableName %>`
-          }
-        }).catch((error) => {
-          console.error(error)
-          alert('There was an error.')
-        })
-      }
+        function handleDeleteClick(event) {
+          event.preventDefault()
+          fetch(document.location.pathname, {
+            method: 'delete'
+          }).then((response) => {
+            if (response.ok) {
+              window.location = `/tables/<%= TableName %>`
+            }
+          }).catch((error) => {
+            console.error(error)
+            alert('There was an error.')
+          })
+        }
       </script>
       <% if (!isNew) { %>
-        <button
-          class='btn btn-danger'
-          onclick='handleDeleteClick(event)'
-          type='button'
-        >
+        <button class='btn btn-danger' onclick='handleDeleteClick(event)' type='button'>
           Delete
         </button>
-      <% } %>
+        <% } %>
     </nav>
     <article id="documentWrapper" style='position: relative; margin-top: 16px; width: 100%; height: 100%'>
-      <div id='documentTextarea' style='
-        position: absolute;
-        top: 0;
-        right: 0;
-        bottom: 0;
-        left: 0;'><%= JSON.stringify(Item, null, 2) %></div>
+      <textarea id='documentTextarea'><%= JSON.stringify(Item, null, 2) %></textarea>
       <script>
-      const documentWrapper = document.getElementById('documentWrapper')
-      function setTextareaHeight (){
-          documentWrapper.style.height = `${window.innerHeight}px`
-      }
-      setTextareaHeight()
+        const editor = CodeMirror.fromTextArea(document.getElementById('documentTextarea'), {
+          // theme: "monokai"
```

Comment:
Can we keep using monokai theme? I kinda liked it more.
Also IMO text is a bit small, can we use something closer to original size used in ace editor? I feel that it's easier to edit code with bigger text.